tests/how_to_test_systems.rs

```diff
@@ -95,3 +111,60 @@ fn spawn_enemy_using_input_resource() {
     // Check resulting changes, no new entity has been spawned
     assert_eq!(world.query::<&Enemy>().iter(&world).len(), 1);
 }
+
+#[test]
+fn confirm_system_is_framerate_independent() {
+    // Setup world
+    let mut world = World::default();
+
+    // Setup stage with a system
+    let mut update_stage = SystemStage::parallel();
+    update_stage.add_system(update_position.system());
+
+    // Closure that gets the resulting position for a certain fps
+    let mut test_fps = |fps: u32| -> Position {
+        // The frame time delta we want to simulate
+        let delta = Duration::from_secs_f32(1.0 / fps as f32);
+
+        // Setup test entities
+        let entity = world
+            .spawn()
+            .insert_bundle((Velocity(vec2(1.0, 0.0)), Position(vec2(0.0, 0.0))))
+            .id();
+
+        // Setup test resource
+        let time = Time::default();
+        world.insert_resource(time);
+
+        // Set initial time
+        let mut time = world.get_resource_mut::<Time>().unwrap();
+        let initial_instant = Instant::now();
+        time.update_with_instant(initial_instant);
+
+        // Simulate one second
+        for i in 0..fps + 1 {
+            // Update time
+            let mut time = world.get_resource_mut::<Time>().unwrap();
+            time.update_with_instant(initial_instant + delta * i);
+
+            // Run systems
+            update_stage.run(&mut world);
+        }
+
+        // Remove mocked Time
+        world.remove_resource::<Time>();
+
+        // Return resulting position
+        *world.get_entity(entity).unwrap().get::<Position>().unwrap()
+    };
+
+    // Test at 30 and 60 fps
+    let result_30fps = test_fps(30);
+    let result_60fps = test_fps(60);
```

Comment:
```suggestion
    let result_15fps = test_fps(15);
    let result_30fps = test_fps(30);
    let result_60fps = test_fps(60);
    let result_120fps = test_fps(120);
    let result_240fps = test_fps(240);
```
Perhaps we should test some more extreme framerates as well.